linux/dev_setup_script.sh

```diff
@@ -0,0 +1,84 @@
+printf "\n\n====> Linux Development Setup <===="
```

Comment:
Actually, it should be called `install_requirements_linux` or something like that.